docs/rules.md

```diff
@@ -101,14 +114,18 @@ Scans a set of markup files for broken links. Links are tested by first renderin
+Note: provide either `content` or `contents`
+
+| Input                    | Required                           | Type       | Default                             | Description                                                                                                                                                                                                                      |
+|--------------------------|------------------------------------|------------| ----------------------------------- |----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `globsAll`               | **Yes**                            | `string[]` |                                     | A list of globs to get files for. This rule passes if none of the files returned by the globs match the supplied string, or if no files are returned.                                                                            |
+| `content`                | **Yes** if `contents` is undefined | `string`   |                                     | The regular expression to check using `String#search`. This expression should not include the enclosing slashes and may need to be escaped to properly integrate with the JSON config (ex. `".+@.+\\..+"` for an email address). |
+| `contents`               | **Yes** if `content` is undefined  | `string[]` |                                     | Regular expressions to check using `String#search`. This expression should not include the enclosing slashes and may need to be escaped to properly integrate with the JSON config (ex. `".+@.+\\..+"` for an email address).    |
+| `nocase`                 | No                                 | `boolean`  | `false`                             | Set to `true` to make the globs case insensitive. This does not effect the case sensitivity of the regular expression.                                                                                                           |
+| `nocase`                 | No                                 | `boolean`  | `false`                             | Set to `true` to make the globs case insensitive. This does not effect the case sensitivity of the regular expression.                                                                                                           |
+| `flags`                  | No                                 | `string`   | `""`                                | The flags to use for the regular expression in `content` (ex. `"i"` for case insensitivity).                                                                                                                                     |
```

Comment:
Is it just intended overlap that someone could set a flags of 'i', or could turn on nocase?

Reply:
`nocase` is for `globsAll` file name search, `flags` is for contents regex search, the usual `gi` part after /.
They are used in different places.